File: soutLink-frontend/src/app/services/models/student-requests-response.ts
/* tslint:disable */
/* eslint-disable */
import { StudentResponse } from '../models/student-response';
export interface StudentRequestsResponse {
  date?: string;
  description?: string;
  id?: number;
  projectTitle?: string;
  status?: 'PENDING' | 'REJECTED' | 'ACCEPTED';
  students?: Array<StudentResponse>;
}

File: soutLink-backend/src/main/java/com/soutLink/supervisor/dto/StudentRequestsResponse.java
package com.soutLink.supervisor.dto;

import com.soutLink.enums.ProjectRequestStatus;
import lombok.*;

import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class StudentRequestsResponse {
    private Long id;
    private String projectTitle;
    private ProjectRequestStatus status;
    private List<StudentResponse> students;
    private String description;
    private String date;
}
